frontend/src/components/Container.js

```diff
@@ -32,6 +33,7 @@ class Container extends PureComponent {
       actions,
       showSidelist,
       siteName,
+      pendingAdvSearch, // indicator flag to show spinner while fetching data for the advanced search
```

Comment:
in Container component, wondering why not calling this variable `showSpinner`?

basically, the container does not care if the spinner is for advanced search or for something else.

Reply:
I find `pendingAdvSearch` has more meaning across the app and in the redux store as it is easier for a developer to differentiate. If a developer opens the reducer and looks for` showSpinner` would not know that the spinner is shown for the case when there is a pending request to search. In case of `pendingAdvSearch` that is self explanatory.